worlds/aquaria/docs/setup_fr.md

```diff
@@ -116,3 +120,15 @@ pour vous assurer que votre fichier est exécutable:
 ```bash
 chmod +x aquaria_randomizer
 ```
+
+## Suivi automatique
+
+Aquaria permet un suivi complètement automatique des cartes du jeu dans le tracker.
+
+1. Téléchargez [Aquaria AP Tracker](https://github.com/palex00/aquaria-ap-tracker/releases/latest) et [PopTracker](https://github.com/black-sliver/PopTracker/releases/latest).
+2. Mettre le paquet du tracker dans le sous-répertoire /packs/ du répertoire d'installation de PopTracker.
+3. Démarrez PopTracker, and chargez le paquet d'Aquaria.
+4. Pour activer le suivi automatique, appuyez sur le symbole "AP" dans le haut de la fenêtre.
+5. Entrez l'adresse du serveur Archipelago (le serveur que votre client s'est connecté), votre nom de connexion et le mot de passe (si un mot de passe est nécessaire).
+
+Le logiciel vous indiquera si une mise à jour du paquet est disponible.
```

Comment:
> Aquaria à un tracker complet qui supporte le tracking automatique.

should be:

Aquaria a un tracker complet qui supporte le tracking automatique.

> Mettre le paquet du tracker dans le sous-répertoire /packs/ du répertoire d'installation de PopTracker.

Maybe it is because of my technical skill, but I still prefer my phrasing.

> Pour activer le tracking automatique, cliquez sur le symbole "AP" en haut de la fenêtre.

I think that "en haut" is more familiar than "dans le haut".

For the rest, I don't have any objection.

Reply:
I think it would be better to call the zip file a fichier.

Reply:
Could someone suggest a complete version?

Reply:
> Could someone suggest a complete version?

Here is what I propose, base on the discussions:

```suggestion
# Tracking automatique

Aquaria a un tracker complet qui supporte le tracking automatique.

1. Téléchargez [Aquaria AP Tracker](https://github.com/palex00/aquaria-ap-tracker/releases/latest) et [PopTracker](https://github.com/black-sliver/PopTracker/releases/latest).
2. Mettre le fichier du tracker dans le sous-répertoire /packs/ du répertoire d'installation de PopTracker.
3. Lancez PopTracker, et ouvrez le pack d'Aquaria.
4. Pour activer le tracking automatique, cliquez sur le symbole "AP" dans le haut de la fenêtre.
5. Entrez l'adresse du serveur Archipelago (le serveur auquel vous avez connecté le client), le nom de votre slot, et le mot de passe (si un mot de passe est nécessaire).

Le logiciel vous indiquera si une mise à jour du pack est disponible.
```